sdk/KoreBuild/scripts/common.sh

```diff
@@ -90,24 +90,23 @@ __fetch() {
 
     __verbose "Downloading $remote_path"
 
-    failed=false
+    local succeeded=false
     if __machine_has 'wget'; then
         # Try wget first as this has been more reliable than curl.
         # Travis CI frequently has TLS issues with curl on macOS
         # Only show progress bar if shell is interactive
         progress_bar='--quiet'
         [ "$__is_verbose" = true ] && [ -z "${PS1:-}" ] && progress_bar='--progress=bar --show-progress'
-        __exec wget $progress_bar --tries 10 -O "$local_path" "$remote_path" || failed=true
+        __exec wget $progress_bar --tries 10 -O "$local_path" "$remote_path" && succeeded=true
```

Comment:
The problem with && is that if wget fails, it won't actually fallback to curl. The first part of this check will fail and cause the script to exit because we have `set -e` at the top.